src/api/index.ts

```diff
@@ -24,6 +24,7 @@ export const URL = {
     GET_PLAYERS: '/api/player/all',
   },
   match: {
+    GET_MATCHES: 'api/match/all',
```

Comment:
it would be great to update this endpoint to be `api/match`. This is just an NTH that should be implemented in the future.

Reply:
We'll add this in a future tkt